projects/configs/tag-operations/test-extract-uclh-omop-cdm.yaml

```diff
@@ -26,7 +26,7 @@
 - name: "SOP Instance UID"
   group: 0x0008
   element: 0x0018
-  op: "hash-uid"
+  op: "replace_UID"
```

Comment:
I was under the impression you were ok with doing this and I was the holdback. It is anyway the users/clients we should ask about this, rather than the devs (as I originally intended to), I think. 

Should I add the custom action that keeps it?

Reply:
Ah I thought we settled on keeping the same, nah I'm fine with it. If we have users who want to put it back into PACS then we can build that out